src/Centreon/Domain/PlatformInformation/PlatformInformation.php

```diff
@@ -225,7 +225,7 @@ public function getApiUsername(): ?string
      */
     public function setApiUsername(?string $username): self
     {
-        $this->apiUsername = $username;
+        $this->apiUsername = filter_var($username, FILTER_SANITIZE_STRING);
```

Comment:
could throw an exception before converting a central to a remote with an empty name ?

Reply:
maybe here : https://github.com/centreon/centreon/pull/9552/files#diff-da80072d5fb9f20be48bea7b7998e32b928914d9c2b59efbefef0c385a40e145R216